index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>

```

Comment:
Also, you need to add `prettier`, `css_comd` to your VScode. I could help you.
Prettier and CssComb will help you have one format in`html`, `css` files.